src/patterns/strategy/strategies.impl.ts

```diff
@@ -0,0 +1,21 @@
+import {IFixedDiscountStrategy, IPercentageDiscountStrategy} from './interfaces';
+
+export class FixedDiscount implements IFixedDiscountStrategy {
```

Comment:
It seems to me that it's better to keep different strategies implementations in different modules